tkltest/generate/unit/augment.py

```diff
@@ -186,7 +186,10 @@ def __compute_base_and_augmenting_tests_coverage(ctd_test_dir, evosuite_test_dir
     # create backup of CTD-guided tests
     ctd_test_dir_bak = os.path.basename(ctd_test_dir) + '-augmentation-bak'
     shutil.rmtree(ctd_test_dir_bak, ignore_errors=True)
-    shutil.move(ctd_test_dir + os.sep + 'monolithic', ctd_test_dir_bak)
+    os.mkdir(ctd_test_dir_bak)
+    for dir in [dir for dir in os.listdir(ctd_test_dir) if os.path.isdir(dir) and dir != 'build' and dir != 'target']:
```

Comment:
At this point are we still running in monolithic? if not, then the check for `build` and `target` is redundant, because they will not be inside the tests directory

Reply:
reminder - In this PR, build directories are in the test directory.
whey will be removed only an the next PR